internal/testserver/datastore/crdb.go

```diff
@@ -19,7 +19,11 @@ import (
 	"github.com/authzed/spicedb/pkg/secrets"
 )
 
-const enableRangefeeds = `SET CLUSTER SETTING kv.rangefeed.enabled = true;`
+const (
+	CRDBTestVersionTag = "v22.2.0"
```

Comment:
Does this mean we'll lose support for v22.1 for watch then?

Reply:
Ugh, good catch. We'll have to add the `min_checkpoint_frequency` flag conditionally based on cockroach version.

`ERROR: invalid option "min_checkpoint_frequency" (SQLSTATE XXUUU).`

Reply:
Well that's going to have to wait.

Reply:
Updated